packages/yoga/src/AutoComplete/web/AutoComplete.test.jsx

```diff
@@ -116,6 +124,33 @@ describe('<AutoComplete />', () => {
       fireEvent.click(getByRole('button'));
 
       expect(onCleanMock).toHaveBeenCalledWith('');
+      expect(onSelectMock).not.toHaveBeenCalled();
+      expect(onChangeMock).toHaveBeenCalled();
```

Comment:
Should we use `toHaveBeenCalledWith` on these specs to assert if we are passing the correct values to the callback functions?

Reply:
yes! well noticed, thanks!